auth_token/utils.py

```diff
@@ -184,7 +187,7 @@ def login(request, user, auth_slug=None, related_objs=None, backend=None, allowe
         allowed_cookie=allowed_cookie,
         allowed_header=allowed_header,
         is_authenticated=not two_factor_login,
-        expires_at=compute_authorization_token_expires_at(expiration),
+        expires_at=compute_authorization_token_expires_at(expiration or get_auth_token_age(request)),
```

Comment:
This one will be better:
```
expiration if expiration is not None else get_auth_token_age(request)
```
Since `0` can be passed as desired value.

Reply:
done